--- w3m-0.5.3/acinclude.m4.orig	2021-01-12 20:37:53.279630897 +0100
+++ w3m-0.5.3/acinclude.m4	2021-01-12 20:54:09.407752277 +0100
@@ -732,6 +732,8 @@
    else
      AC_MSG_WARN([unable to build w3mimgdisplay with X11 support])
    fi
+   IMGX11CFLAGS="$IMGX11CFLAGS `${PKG_CONFIG} --cflags x11`"
+   IMGX11LDFLAGS="$IMGX11LDFLAGS `${PKG_CONFIG} --libs x11`"
   fi
   if test x"$fb" = xyes; then
    if test x"$have_gtk2" = xyes; then
--- w3m-0.5.3/configure.ac.orig	2021-01-12 20:55:05.160780806 +0100
+++ w3m-0.5.3/configure.ac	2021-01-12 21:01:52.818543811 +0100
@@ -56,6 +56,7 @@
 AC_W3M_ANSI_COLOR
 AC_W3M_BG_COLOR
 
+PKG_PROG_PKG_CONFIG
 AC_W3M_IMAGE
 if test x"$enable_image" != xno; then
  AUXBIN_TARGETS="$AUXBIN_TARGETS w3mimgdisplay\$(EXT)"
