? w3m-0.3.2-security.patch
Index: frame.c
===================================================================
RCS file: /cvsroot/w3m/w3m/frame.c,v
retrieving revision 1.16
retrieving revision 1.16.4.1
diff -u -r1.16 -r1.16.4.1
--- frame.c	17 Jun 2002 15:50:45 -0000	1.16
+++ frame.c	26 Nov 2002 07:11:22 -0000	1.16.4.1
@@ -1,4 +1,4 @@
-/* $Id$ */
+/* $Id$ */
 #include "fm.h"
 #include "parsetagx.h"
 #include "myctype.h"
@@ -519,14 +519,15 @@
 		    frame.body->attr = F_UNLOADED;
 		    if (frame.body->flags & FB_NO_BUFFER)
 			fprintf(f1, "Open %s with other method",
-				frame.body->url);
+				html_quote(frame.body->url));
 		    else if (frame.body->url)
-			fprintf(f1, "Can't open %s", frame.body->url);
+			fprintf(f1, "Can't open %s",
+				html_quote(frame.body->url));
 		    else
 			fprintf(f1,
 				"This frame (%s) contains no src attribute",
-				frame.body->name ? frame.body->
-				name : "(no name)");
+				frame.body->name ? html_quote(frame.body->name)
+				: "(no name)");
 		    break;
 		}
 		parseURL2(frame.body->url, &base, currentURL);
@@ -620,7 +621,7 @@
 			    /* prohibit_tags */
 			    Strshrinkfirst(tok, 1);
 			    Strshrink(tok, 1);
-			    fprintf(f1, "<!-- %s -->", tok->ptr);
+			    fprintf(f1, "<!-- %s -->", html_quote(tok->ptr));
 			    goto token_end;
 			case HTML_TABLE:
 			    t_stack++;
@@ -633,7 +634,7 @@
 				Strshrink(tok, 1);
 				fprintf(f1,
 					"<!-- table stack underflow: %s -->",
-					tok->ptr);
+					html_quote(tok->ptr));
 				goto token_end;
 			    }
 			    break;
@@ -653,7 +654,8 @@
 			    if (!t_stack) {
 				Strshrinkfirst(tok, 1);
 				Strshrink(tok, 1);
-				fprintf(f1, "<!-- %s -->", tok->ptr);
+				fprintf(f1, "<!-- %s -->",
+					html_quote(tok->ptr));
 				goto token_end;
 
 			    }
 
