--- w3m-0.2.0/rc.c.orig	Thu Mar 22 08:42:52 2001
+++ w3m-0.2.0/rc.c	Wed Apr  4 22:10:53 2001
@@ -214,7 +214,7 @@
 #endif				/* defined(USE_SSL) &&
 				 * defined(USE_SSL_VERIFY) */
 #ifdef USE_SSL
-#define CMT_SSL_FORBID_METHOD N_("List of forbidden SSL method (2: SSLv2, 3: SSLv3, t:TLSv1)")
+#define CMT_SSL_FORBID_METHOD "List of forbidden SSL method (2: SSLv2, 3: SSLv3, t:TLSv1)"
 #endif
 #ifdef USE_COOKIE
 #define CMT_USECOOKIE   "Use Cookie"
