diff -Nur w3m-0.2.3.2.orig/url.c w3m-0.2.3.2/url.c
--- w3m-0.2.3.2.orig/url.c	Fri Dec  7 08:58:07 2001
+++ w3m-0.2.3.2/url.c	Sun Dec 23 15:22:48 2001
@@ -239,6 +239,7 @@
 {
     if (ssl_ctx != NULL)
 	SSL_CTX_free(ssl_ctx);
+    ssl_ctx = NULL;
 }
 
 #if SSLEAY_VERSION_NUMBER >= 0x00905100
@@ -288,7 +289,6 @@
 	ssl_path_modified = 1;
 #else
 	free_ssl_ctx();
-	ssl_ctx = NULL;
 #endif
     }
 #ifdef USE_SSL_VERIFY
@@ -298,7 +298,6 @@
     }
     if (ssl_path_modified) {
 	free_ssl_ctx();
-	ssl_ctx = NULL;
 	ssl_path_modified = 0;
     }
 #endif				/* defined(USE_SSL_VERIFY) */
@@ -350,9 +349,6 @@
 #endif				/* defined(USE_SSL_VERIFY) */
 	    SSL_CTX_set_default_verify_paths(ssl_ctx);
 #endif				/* SSLEAY_VERSION_NUMBER >= 0x0800 */
-/*** by inu
-        atexit(free_ssl_ctx);
-*/
     }
     handle = SSL_new(ssl_ctx);
     SSL_set_fd(handle, sock);
